mln/admin/__init__.py

```diff
@@ -29,7 +29,7 @@ class FriendshipAdmin(admin.ModelAdmin):
 
 custom[Friendship] = FriendshipAdmin
 
-has_trigger = lambda obj: NetworkerMessageTrigger.objects.filter(body=obj).exists() or NetworkerFriendshipCondition.objects.filter(Q(success_body=obj) | Q(failure_body=obj)).exists()
+has_trigger = lambda obj: NetworkerMessageTrigger.objects.filter(response=obj).exists() or NetworkerFriendshipCondition.objects.filter(Q(success_body=obj) | Q(failure_body=obj)).exists()

```

Comment:
If you're keeping TriggerDev around you should also keep this field around, and just add another for checking actual triggers.